resources/webroot/static/data/en.json

```diff
@@ -252,27 +339,45 @@
         "stats_male_spawns": "male spawns",
         "stats_female_spawns": "female spawns",
         "stats_sex": "Sex",
-        "stats_100iv": "100% IV:",
-        "stats_90iv": "90% IV:",
-        "stats_0iv": "0% IV:",
+        "stats_100iv": "100% IV",
+        "stats_90iv": "90% IV",
+        "stats_0iv": "0% IV",
+        "stats_lifetime": "Lifetime",
+        "stats_30days": "30 days",
+        "stats_today": "Today",
+        "stats_iv_today": "IV Today",
         "stats_date": "Date",
         "stats_eggs": "Eggs",
         "stats_raids": "Raids",
         "stats_quests": "Field Research Quests",
         "stats_quests_item_rewards": "Item Rewards",
-        "stats_quests_pokemon_rewards": "Pokemon Rewards",
+        "stats_quests_pokemon_rewards": "Pokémon Rewards",
         "stats_grunt_types": "Grunt Types",
         "stats_statistics": "Statistics",
-        "stats_active_100iv": "100% IV Pokemon",
-        "stats_active_90iv": "90% IV Pokemon",
-        "stats_active_0iv": "0% IV Pokemon",
+        "stats_active_100iv": "100% IV Pokémon",
+        "stats_active_90iv": "90% IV Pokémon",
+        "stats_active_0iv": "0% IV Pokémon",
         "stats_total_shiny": "Total Shiny",
         "stats_total_hundo": "Total 100% IV",
-        "stats_active_shiny": "Shiny Pokemon",
+        "stats_active_shiny": "Shiny Pokémon",
         "stats_active_iv_statistics": "Active IV Statistics",
-        "stats_top10_pokemon": "Top 10 Pokemon",
+        "stats_top10_pokemon": "Top 10 Pokémon",
         "stats_filters": "Filters",
         "stats_new_pokestops": "New Pokestops (Last 24 Hours)",
-        "stats_new_gyms": "New Gyms (Last 24 Hours)"
+        "stats_new_gyms": "New Gyms (Last 24 Hours)",
+        "stats_historic_pokemon": "Historical Pokémon Statistics",
+        "stats_historic_raid": "Historical Raid Statistics",
+        "stats_historic_quests": "Historical Quest Statistics",
+        "stats_historic_invasion": "Historical Invasion Statistics",
+        "weather_types": {
```

Comment:
add `"weather_0": "Extreme"` etc. here instead

Reply:
It's already part of external translations from pogo-translations, can be used out of the box, same as e.g. `poke_type_1`. Good idea :)